### PR TITLE
Support for API1000 - Logical Interconnect Group

### PR DESCRIPTION
**Pull request by jyothis-gireesan**
### Description
- Adds support for API1000 - Logical Interconnect Group
### Check List
- [x] New functionality includes testing.
  - [x] All tests pass for go 1.11.x + gofmt checks .
- [ ] New functionality has been documented in the README if applicable.
  - [X] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [X] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/resource_logical_interconnect_group.go [hunk 1]
```diff
@@ -597,9 +775,159 @@ func resourceLogicalInterconnectGroupCreate(d *schema.ResourceData, meta interfa
...
+	if val, ok := d.GetOk(sflowConfigurationPrefix + ".status"); ok {
+		sflowConfiguration.Status = val.(string)
+	}
+	if val, ok := d.GetOk(sflowConfigurationPrefix + ".uri"); ok {
+		sflowConfiguration.URI = utils.NewNstring(val.(string))
+	}
+
+	if val, ok := d.GetOk(sflowConfigurationPrefix + ".type"); ok {
+		sflowConfiguration.Type = val.(string)
+		lig.SflowConfiguration = &sflowConfiguration
```
Comment (madhav-bharadwaj) [inline oneview/resource_logical_interconnect_group.go]: Type being an optional attribute, Is the non-assignment in line928 expected if type value is not provided?

File: oneview/resource_logical_interconnect_group.go [hunk 2]
```diff
@@ -274,6 +280,163 @@ func resourceLogicalInterconnectGroup() *schema.Resource {
 					},
 				},
 			},
+			"sflow_configuration": {
+				Type:     schema.TypeList,
```
Comment (madhav-bharadwaj) [inline oneview/resource_logical_interconnect_group.go]: Is TypeList required for this attribute? Can you point me to the documentation for this attribute. I'm not able to find it.

File: oneview/resource_logical_interconnect_group.go (hunk 1 above)
Comment (jyothis-gireesan) [reply oneview/resource_logical_interconnect_group.go]: i gave type as optional so the value can be empty. It is mandatory if this struct is passed.

File: oneview/resource_logical_interconnect_group.go (hunk 2 above)
Comment (jyothis-gireesan) [reply oneview/resource_logical_interconnect_group.go]: its in 1000 and above versions for LIG

**Pull request merged**